assets/js/components/Dashboard/PermanentConfiguration/SelectScreenType.tsx

```diff
@@ -61,8 +61,8 @@ const SelectScreenTypeComponent: ComponentType = () => {
         <Col>
           <ButtonImage
             fileName="solari.png"
-            label="Solari"
-            onClick={() => selectScreenType("solari")}
+            label="Sectional"
+            onClick={() => selectScreenType("sectional")}
```

Comment:
Note: This isn't hooked up to anything yet, so nothing else needs to change to accommodate the changed identifier here.